There probably shouldn't be a test.only() used at /test/unit/io/files.js line 469

So I was going through the code for the tests and felt that this was important to bring to the attention of the maintainers. 
There probably shouldn't be a `test.only()` used [here](https://github.com/processing/p5.js/blob/65ed7a9b12f04a0efdbb6edc8766b9803a01254a/test/unit/io/files.js#L469)

I don't fully understand the tests yet so I am sorry if this ends up being stupid.

Unfortunately, that doesn't seem to be the case. When I remove `.only()` and try `npm test` on the current code on master, it runs a whole lot more test cases that didn't run before. 
2 of them fail
This is in the "mochaChrome:test" (mochaChrome) task
 ```
 1378 passing (14s)
  10 pending
  2 failing
  1) Error Helpers
       friendly error logger
         basic:
     AssertionError: got unwanted exception: expected [Function] to not throw 'Error' but 'TypeError: Cannot read property \'substring\' of undefined' was thrown
      at Function.assert.doesNotThrow (http://localhost:9001/node_modules/chai/chai.js:3300:35)
      at Context.<anonymous> (unit/core/error_helpers.js:19:14)

  2) DOM
       p5.Element.prototype.position
         time and date
           p5.prototype.millis
             result should be greater than running time:
     AssertionError: everything is ok: expected 49.689999999827705 to be > 50
  

Warning: [object Object] Use --force to continue.

Aborted due to warnings.
npm ERR! Test failed.  See above for more details.

```

@stalgiag @outofambit can you please confirm ?

This is the output with the .only() in place, as is in master
```
Running "mochaChrome:test" (mochaChrome) task

  DOM
    p5.Element.prototype.position
      Files
        p5.prototype.saveStrings
          ✓ should be a function
  1 passing (9ms)

  DOM
    p5.Element.prototype.position
      Files
        p5.prototype.saveStrings
          ✓ should be a function
  1 passing (8ms)

Running "mochaTest:test" (mochaTest) task
 8   -_-_-_-_-_,------,
 0   -_-_-_-_-_|   /\_/\ 
 0   -_-_-_-_-^|__( ^ .^) 
     -_-_-_-_-  ""  "" 

  8 passing (16ms)
```

--- a/src/core/error_helpers.js
+++ b/src/core/error_helpers.js
@@ -805,13 +805,21 @@ const helpForMisusedAtTopLevelCode = (e, log) => {
     if (e.message && e.message.match(`\\W?${symbol.name}\\W`) !== null) {
       const symbolName =
         symbol.type === 'function' ? `${symbol.name}()` : symbol.name;
-      log(
-        translator('fes.misusedTopLevel', {
-          symbolName,
-          symbolType: symbol.type,
-          link: FAQ_URL
-        })
-      );
+      if (typeof IS_MINIFIED !== 'undefined') {
+        log(
+          `Did you just try to use p5.js's ${symbolName} ${
+            symbol.type
+          }? If so, you may want to move it into your sketch's setup() function.\n\nFor more details, see: ${FAQ_URL}`
+        );
+      } else {
+        log(
+          translator('fes.misusedTopLevel', {
+            symbolName,
+            symbolType: symbol.type,
+            link: FAQ_URL
+          })
+        );
+      }
       return true;
     }
   });

--- a/test/unit/core/error_helpers.js
+++ b/test/unit/core/error_helpers.js
@@ -18,7 +18,7 @@ suite('Error Helpers', function() {
     test('basic', function() {
       assert.doesNotThrow(
         function() {
-          p5._friendlyError('basic');
+          p5._friendlyError('basic', 'basic');
         },
         Error,
         'got unwanted exception'

--- a/test/unit/io/files.js
+++ b/test/unit/io/files.js
@@ -466,7 +466,7 @@ suite('Files', function() {
 
   // saveStrings()
   suite('p5.prototype.saveStrings', function() {
-    test.only('should be a function', function() {
+    test('should be a function', function() {
       assert.ok(myp5.saveStrings);
       assert.typeOf(myp5.saveStrings, 'function');
     });

--- a/test/unit/utilities/time_date.js
+++ b/test/unit/utilities/time_date.js
@@ -108,9 +108,9 @@ suite('time and date', function() {
 
     test('result should be greater than running time', function() {
       var runningTime = 50;
-      var init_date = Date.now();
+      var init_date = window.performance.now();
       // wait :\
-      while (Date.now() - init_date < runningTime) {
+      while (window.performance.now() - init_date < runningTime) {
         /* no-op */
       }
       assert.operator(myp5.millis(), '>', runningTime, 'everything is ok');

